docs/installing.md

```diff
@@ -138,8 +138,8 @@ and see instructions for configuring Element. If you don't, your configuration i
 intended - double check that all the configuration is set up and visit [#dimension:t2bot.io](https://matrix.to/#/#dimension:t2bot.io)
 for further help.
 
-After configuring Element, click the integrations button (4 squares in the top right of any room) and
```

Comment:
Maybe just a nitpick, but the mobile clients on Android and iOS still have the integrations button on the top right. I think it makes sense to explain both variants.